server/src/board/storage.rs

```diff
@@ -21,6 +22,7 @@ impl Service {
             id,
             chat: Default::default(),
             lines: Default::default(),
+            user_positions: Default::default(),
```

Comment:
as user positions are not apart of the board state - do you think we need to save them?

Reply:
We could save user positions temporarily so that whenever a new user joins the board, we can send all user positions as part of the state however we don't need to do that as you said.